Source/WebKit/UIProcess/Cocoa/VideoFullscreenManagerProxy.mm

```diff
@@ -1137,6 +1146,21 @@ - (BOOL)prefersStatusBarHidden
     m_page->send(Messages::VideoFullscreenManager::FullscreenMayReturnToInline(contextId, m_page->isViewVisible()));
 }
 
+#if HAVE(UI_WINDOW_SCENE_LIVE_RESIZE)
+
+void VideoFullscreenManagerProxy::callAfterPlayerLayerBoundsHaveChanged(CompletionHandler<void()>&& callback)
+{
+    m_playerLayerBoundsHaveChangedCallbacks.append(WTFMove(callback));
+}
+
+void VideoFullscreenManagerProxy::didResolvePlayerLayerBounds()
```

Comment:
I think I need to somehow ensure these callbacks will always get called, but not sure if I should put a call to this in the destructor or elsewhere?